pyqtgraph/__init__.py

```diff
@@ -378,8 +378,6 @@ def exit():
 
 ## Convenience functions for command-line use
 
-plots = []
```

Comment:
These references should be necessary to ensure the plots stay alive as long as they the user keeps them open. Perhaps we can have the references automatically removed when the window is closed?